tools/diff-processor/detector/detector.go

```diff
@@ -71,15 +101,13 @@ func markCoverage(fieldCoverage ResourceChanges, config reader.Resource) error {
 	return nil
 }
 
-func untestedFields(fieldCoverage ResourceChanges, path []string) []string {
+func untestedFields(fieldCoverage ResourceChanges) []string {
 	fields := make([]string, 0)
-	for fieldName, coverage := range fieldCoverage {
+	for key, coverage := range fieldCoverage {
 		if field, ok := coverage.(*Field); ok {
 			if !field.Tested {
-				fields = append(fields, strings.Join(append(path, fieldName), "."))
+				fields = append(fields, key)
 			}
-		} else if objectCoverage, ok := coverage.(ResourceChanges); ok {
```

Comment:
why is this no longer necessary?

Reply:
The fields are now flattened, so there is no need to handle objects here.

Reply:
ack, thanks!